src/sentry/tasks/integrations/github/pr_comment.py

```diff
@@ -62,3 +65,29 @@ def pr_to_issue_query():
             [GroupOwnerType.SUSPECT_COMMIT.value],
         )
         return cursor.fetchall()
+
+
+def get_top_5_issues_by_count(issue_list: List[int]) -> List[int]:
+    """Given a list of issue group ids, return a sublist of the top 5 ordered by event count"""
+    request = SnubaRequest(
+        dataset=Dataset.Events.value,
+        app_id="default",
+        query=(
+            Query(Entity("events"))
```

Comment:
This query is going to fail because it needs [tenant ids](https://vanguard.getsentry.net/p/clgxyod9m0000s60lazmrg8j3) 

CI should reject it as well, Is the test running queries against a snuba image or a mock?

Reply:
Is there a way to mock the dataset instead to make sure we're testing the query? I understand it wouldn't catch the problem of tenant ids here but it would catch the logic issues if any.